content_scripts.js

```diff
@@ -1615,6 +1615,11 @@ Normal.insertJS = function(url) {
     s.src = url;
     document.lastElementChild.appendChild(s);
 };
+Normal.moveTab = function(pos) {
+    RUNTIME('moveTab', {
+        position: pos
+    });
+}
```

Comment:
And please append `;` at line 1622 to make sure that js-beautify will help us to beautify the code.